docs/configuration/git-providers/bitbucket-enterprise.md

```diff
@@ -1,15 +1,24 @@
-# Bitbucket Stash
+# Bitbucket Server
 
 Set your configuration values for your Bitbucket instance on the `values.yaml` file.
+
+NOTE: Since bitbucket server uses OAuth1, you'll need to create a key pair to sign and validate the requests between codacy and the bitbucket server instance:
+1. Create the key pair by running the command below, making sure that you don't define a passphrase:
+   
+   `ssh-keygen -t rsa -f mykey -q -N ""`
+   
+1. Copy the private key from the `mykey` file and set it in the `consumerPrivateKey`.
+1. Copy the public key from the `mykey.pub` file and set it in the `consumerPublicKey`. 
```

Comment:
remark-lint formater should put this as `1.` and `2.`

Reply:
I don't agree too much with this rule, it makes it a bit harder to author / maintain the Markdown files - if you add, remove, or reorder items in the list you should also reorder the numbering (even if the source numbering doesn't really affect the rendering on common Markdown processors).